### ISSUE TITLE
time flow is not consistant with actual time flow on speed 1

### ISSUE THREAD

**Issue by Redd56** [dymanoid/RealTime#293]
Ive been doing a ton of tests as i have a project idea and ive noticed that at speed one in the config for the mod time still moves too fast, ive had to slow down time a touch with the game speed slider mod, down to 80 percent, though that is slightly too slow.

**Comment by dymanoid** [comment]
Hi! Thanks for your feedback. There was no intention to make the speed setting "1" in Real Time exactly match the real time flow. Those speed settings are based on the game's engine implementation, so their actual pace is kind of random. Do you see any benefits in matching the speed setting 1 to the real world time? Many players complain that this speed setting is too slow anyway.

**Comment by Redd56** [comment]
Yeah some players may want to actualy run it in true real time (me as an example) for various reason. The reason i want to is a 24/7 realtime cities skylines stream i wanted to do.
